pkg/resource/aws/testdata/acc/aws_route_table_association/.terraform.lock.hcl

```diff
@@ -0,0 +1,20 @@
+# This file is maintained automatically by "terraform init".
```

Comment:
does this belong in the pr ? ^^

Reply:
Yes it is, it't the dependency lock file for acceptance tests

Reply:
Ok got it :+1: